playbooks/roles/dcnm_image_upgrade/dcnm_hosts.yaml

```diff
@@ -0,0 +1,15 @@
+all:
```

Comment:
These diffs are for image_upgrade?  Did you mean this to be part of the fabric v2 diffs?

Reply:
Looks like you have more diffs below related to image upgrade and management.  Just making sure they are supposed to be part of these diffs

Reply:
> Looks like you have more diffs below related to image upgrade and management. Just making sure they are supposed to be part of these diffs. Maybe you need to merge develop into this branch again?

Hi Mike, these diffs were needed for a couple reasons (given the changes are all around the new "SmartAPI" mechanism and also removing dependencies on AnsibleModule (hiding this within Sender() class).

I think this should merge fine as-is.